test/options.jl

```diff
@@ -47,3 +47,10 @@ MathProgBase.loadproblem!(m, -obj, A, rowub, [(:NonNeg,1:3)],[(:NonNeg,1:5)])
 MathProgBase.setwarmstart!(m, primal_sol; dual_sol = dual_sol, slack = slack)
 MathProgBase.optimize!(m)
 @test isapprox(MathProgBase.getobjval(m), -99.0, atol=1e-10, rtol=0.0)
+
+# tests for incorrect options
+s = SCSSolver(eps=1e-12, epps=1.0)
+m = MathProgBase.ConicModel(s)
+MathProgBase.loadproblem!(m, -obj, A, rowub, [(:NonNeg,1:3)],[(:NonNeg,1:5)])
+
+@test_throws ArgumentError MathProgBase.optimize!(m)
```

Comment:
Is there some way to test the text of the error message? That's pretty important in this case.

Reply:
https://github.com/JuliaLang/julia/blob/e7d15d4a013a43442b75ba4e477382804fa4ac49/stdlib/Test/src/Test.jl#L532
`@test_throws` wants type, not an instance

Reply:
Yes, `@test_throws` is probably not the way to do it. It may need a `try/catch` where you test the error message inside the `catch` and test that the `catch` block executed.